ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -247,34 +247,38 @@ type NetworkApps peer =
+  { rnaIpSubscriptionTracer    :: Tracer IO (WithIPList (SubscriptionTrace Socket.SockAddr))
     -- ^ IP subscription tracer
-  , rnaDnsSubscriptionTracer :: Tracer IO (WithDomainName (SubscriptionTrace Socket.SockAddr))
-  , rnaMuxTracer             :: Tracer IO (WithMuxBearer (MuxTrace NodeToNodeProtocols))
+  , rnaDnsSubscriptionTracer   :: Tracer IO (WithDomainName (SubscriptionTrace Socket.SockAddr))
+  , rnaMuxTracer               :: Tracer IO (WithMuxBearer (MuxTrace NodeToNodeProtocols))
     -- ^ Mux tracer
-  , rnaMuxLocalTracer        :: Tracer IO (WithMuxBearer (MuxTrace NodeToClientProtocols))
-  , rnaHandshakeTracer       :: Tracer IO (TraceSendRecv
-                                            (Handshake NodeToNodeVersion CBOR.Term)
-                                            peer
-                                            (DecoderFailureOrTooMuchInput CBOR.DeserialiseFailure))
+  , rnaMuxLocalTracer          :: Tracer IO (WithMuxBearer (MuxTrace NodeToClientProtocols))
+  , rnaHandshakeTracer         :: Tracer IO (TraceSendRecv
+                                              (Handshake NodeToNodeVersion CBOR.Term)
+                                              peer
+                                              (DecoderFailureOrTooMuchInput CBOR.DeserialiseFailure))
     -- ^ Handshake protocol tracer
-  , rnaHandshakeLocalTracer  :: Tracer IO (TraceSendRecv
-                                            (Handshake NodeToClientVersion CBOR.Term)
-                                            peer
-                                            (DecoderFailureOrTooMuchInput CBOR.DeserialiseFailure))
+  , rnaHandshakeLocalTracer    :: Tracer IO (TraceSendRecv
+                                              (Handshake NodeToClientVersion CBOR.Term)
+                                              peer
+                                              (DecoderFailureOrTooMuchInput CBOR.DeserialiseFailure))
     -- ^ DNS subscription tracer
-  , rnaDnsResolverTracer     :: Tracer IO (WithDomainName DnsTrace)
+  , rnaDnsResolverTracer       :: Tracer IO (WithDomainName DnsTrace)
     -- ^ DNS resolver tracer
-  , rnaMkPeer                :: SockAddr -> SockAddr -> peer
+  , rnaMkPeer                  :: SockAddr -> SockAddr -> peer
     -- ^ How to create a peer
-  , rnaMyAddrs               :: [AddrInfo]
+  , rnaMyAddrs                 :: [AddrInfo]
     -- ^ The node's own addresses
-  , rnaMyLocalAddr           :: AddrInfo
+  , rnaMyLocalAddr             :: AddrInfo
     -- ^ The node's own local address
-  , rnaIpProducers           :: [SockAddr]
+  , rnaIpProducers             :: [SockAddr]
     -- ^ IP producers
-  , rnaDnsProducers          :: [DnsSubscriptionTarget]
+  , rnaDnsProducers            :: [DnsSubscriptionTarget]
     -- ^ DNS producers
+  , rnaNodeToNodeVersionData   :: NodeToNodeVersionData
+    -- ^ Node-to-Node Version Data
+  , rnaNodeToClientVersionData :: NodeToClientVersionData
+    -- ^ Node-to-CLient Version Data
```

Comment:
Are `rnaNodeTo{Node,Client}VersionData` used? The patch is smaller than I expect.

Reply:
Oops :man_facepalming:. I force-pushed the remainder of the patch.

Reply:
> Nitpick: should probably be `Node-to-Client` rather than `Node-to-CLient` smile

Fixed, thanks!